package-system/OpenEXR/build_config.json

```diff
@@ -0,0 +1,119 @@
+{
+   "git_url":"https://github.com/AcademySoftwareFoundation/openexr.git",
+   "git_tag":"RB-2.3",
+   "package_name":"ilmbase",
+   "package_version":"2.3.0-rev4",
+   "package_url":"https://github.com/03de/OpenEXR",
+   "package_license":"Apache-2.0",
+   "package_license_file":"LICENSE",
+   "cmake_find_target":"Findilmbase.cmake",
+   "Platforms":{
+      "Windows":{
+         "Windows":{
+            "cmake_find_template":"Findilmbase.cmake.Windows",
+            "cmake_generate_args_debug":[
+               "-G",
+               "\"Visual Studio 16 2019\"",
+               "-A",
+               "x64",
+               "-T",
+               "host=x64",
```

Comment:
The Architecture and toolset argument isn't needed when building with VS2019 and above

Reply:
Will do, removed now.